topics/variant-analysis/tutorials/vir_var0/tutorial.md

```diff
@@ -0,0 +1,618 @@
+
+```
+https://zenodo.org/record/3519268/files/
+```
+
+and the following prefix:
+
+```
+.fq.gz
+```
+
+so that the full URL looks something like this:
+
+```
+https://zenodo.org/record/3519268/files/SRR8525909.fq.gz
+```
+
+But we need to do this for all four datasets (or all 68, or a few thousand if you analyze other data). To help with this we use **Rule builder** - a part of Galaxy's dataset upload functionality.
+
+To understand how Rule Builder operates it is best to watch the following video
+
+```
+VIDEO PLACEHOLDER
+```
+
+Now that you've watched the video, do it yourself:
+
+> ### {% icon hands_on %} Upload with Rule Builder 
+>
+> 1. Click the upload icon {% icon galaxy-upload %} in the upper left corner of the Galaxy interface
+> 2. Select **Rule-based** tab
+> 3. In **Upload data as** dropdown select *Collection(s)*
+> 4. Copy and paste the four dataset IDs listed above into the large text box
+> 5. Click **Build** button
+> 6. Follow step from the video to create a collection
+>
+> > ### {% icon tip %} Tip: JSON rules
+> >
+> > Alternatively you import the following JSON block as shown here **TODO**:
+> >
+> > ```
+> > {
+  "rules": [
+    {
+      "type": "add_column_value",
+      "value": "https://zenodo.org/record/3519268/files/"
+    },
+    {
+      "type": "add_column_concatenate",
+      "target_column_0": 1,
+      "target_column_1": 0
+    },
+    {
+      "type": "remove_columns",
+      "target_columns": [
+        1
+      ]
+    },
+    {
+      "type": "add_column_value",
+      "value": ".fq.gz"
+    },
+    {
+      "type": "add_column_concatenate",
+      "target_column_0": 1,
+      "target_column_1": 2
+    },
+    {
+      "type": "remove_columns",
+      "target_columns": [
+        1,
+        2
+      ]
+    }
+  ],
+  "mapping": [
+    {
+      "type": "list_identifiers",
+      "columns": [
+        0
+      ],
+      "editing": false
+    },
+    {
+      "type": "url",
+      "columns": [
+        1
+      ]
+    }
+  ],
+  "extension": "fastqsanger.gz"
+}
+> > ```
+> {: .tip}
+{: .hands_on}
+
+Once the collection is created and uploaded, you will see a new green box in the history pane as shown in Fig. [2](#figure-2) below:
+
+![Dataset collection](../../images/hiv_collection.png "HIV datasets uploaded as a collection. In this case the collection contains four datasets"){: width="100px"}
+
+# Quality control
+
+## Assessing read quality
+
+Once the data is we will need to assess its quality. There is a number of tutorials here already covering this [subject]({{ site.baseurl }}{% link topics/sequence-analysis/index.md %}), but for continuity we go through QC steps anyway.
+
+> ### {% icon hands_on %} QC: Assessing read quality
+>
+> 1. **FastQC** {% icon tool %} with the following parameters:
+>    - {% icon param-collection %} *"Short read data from your current history"*: `Jair data` (or whatever other name you gave that collection; red arrow in Fig. [3](#figure-3))
+>    - Click **Execute** button
```

Comment:
```suggestion
```

I think all of the execute instructions can be removed? We've removed them from every other hands on box.